module_utils/storage_lsr/argument_validator.py

```diff
@@ -0,0 +1,171 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+class ArgValidator(object):
+
+    @classmethod
+    def _validate_list(cls, value):
+        if isinstance(value, list):
+            return value
+
+        if isinstance(value, str):
```

Comment:
It is a bit complicated to test if an object is a string and make it portable for both python2 and python3
see https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/utils.py#L20-L21
and
https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/utils.py#L280

Reply:
Thank you. Fixed.